app/views/posts/index.html.erb

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+   <ul>
+    <li class='user'>
+        <div class='photo'>
+            <%= image_tag (@user.photo)%>
+        </div>
+        <div class='user-info'>
+             <h2><%= @user.name %></h2>
+             <p>Number of Posts: <%= @user.posts_counter %></p>
+        </div>
+    </li>
+
+        <% @posts.each do |post| %>
+          <li>
+            <% @users.each do |user| %>
+              <% if post.author_id == user.id %>
+                <h4>Post #<%=post.id %> by <%= user.name  %></h4>
+                 <div>
+                  <p>Comments: <%= post.comments_counter %></p>
+                  <p>Likes: <%= post.likes_counter %></p>
+                 </div>
+                 <div>
+    <% post.comments.each do |comment| %>
+        <% if comment.author_id == user.id %>
+        <%= user.name %>:<%= comment.text %> <br>
+        <% end %>
+      <% end %>
```

Comment:
- [ ] Kindly consider rendering 5 most recent comments by using `recent_comments` method in the post model and also consider following design guidelines for the [index](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=GHSAT0AAAAAACBXQZYXKV5PPM2GFX2GYITAZEVU2YA), [show](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_single_post.png?token=GHSAT0AAAAAACBXQZYWZGBQD6GWRBKOV3EGZEVU4TA) post

Reply:
Well resolved ✔️